README.md

```diff
@@ -178,6 +181,43 @@ are not sufficient, you can provide a custom GitHub token with the necessary per
     github-token: ${{ secrets.CUSTOM_GITHUB_TOKEN }}
 ```
 
+### UV_TOOL_DIR
+
+On Windows `UV_TOOL_DIR` is set to `uv-tool-dir` in the `TMP` dir (e.g. `D:\a\_temp\uv-tool-dir`).
+On GitHub hosted runners this is on the much faster `D:` drive.
```

Comment:
So the cache is also on the `D:` drive, is that right?

Reply:
Yes. It's also in the TMP dir so `D:\a\_temp\` on Windows

Reply:
Added documentation for the default value of `UV_CACHE_DIR`